packages/contracts/contracts/Unirep.sol

```diff
@@ -327,63 +359,6 @@ contract Unirep is IUnirep, zkSNARKHelper, VerifySignature {
         proofIndex++;
```

Comment:
```suggestion
        proofIndex = _proofIndex + 1;
```

Reply:
We use the increment operator throughout the contract. I think doing something like `uint someVariable = proofIndex++` is bad/hard to understand, but just using it for incrementing a value seems okay.